cypress/features/regression/buttonAsSiblingClassic.feature

```diff
@@ -0,0 +1,58 @@
+Feature: Button as a sibling classic component
+  I want to check Button as a sibling classic component properties
+
+  Background: Open Button as a sibling component page classic
+    Given I open "Button" classic component for classic story as sibling in no iframe
+
+  @positive
+  Scenario Outline: Set Button size to <size>
+    When I select size to "<size>"
+    Then Button as a sibling height is "<height>" and width is "<width>"
+    Examples:
+      | size   | height | width       |
+      | small  | 25px   | 107.90625px |
+      | medium | 31px   | 138.4375px  |
+      | large  | 43px   | 142.4375px  |
+
+  @positive
+  Scenario Outline: Set Button theme to <theme> and as to primary
+    When I select theme to "<theme>"
+      And I select as to "primary"
```

Comment:
This need to be as Given and moved up

Reply:
fix

Reply:
`Given` I select as to "primary"
`When` I select theme to "<theme>"